Integrations/integration-AbuseDB.yml

```diff
@@ -107,7 +108,7 @@ script:
         21 : 'Web_App_Attack',
         22 : 'SSH',
         23 : 'IoT_Targeted'
-    }
+    }script-AbuseIPDB-PopulateIndicators.yml
```

Comment:
Not sure why you have here this yml file. Possibly a typo?

Reply:
yeah, thanks.